### PR TITLE
config-tools: update board inspector to update pci.ids before fetchin…

### PR DESCRIPTION
**Pull request by jackwhich**
…g PCI information

Add lspci -q Query PCI ID database via DNS for unknown ID‘s

Tracked-On: #7086
Signed-off-by: zhongzhenx.liu <zhongzhenx.liu@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: misc/config_tools/board_inspector/board_inspector.py [hunk 1]
```diff
@@ -40,6 +40,16 @@ def check_deps():
                 logging.warning("This tool requires CPUID version >= {}".format(cpuid_min_ver))
                 sys.exit(1)
 
+    # Try updating pci.ids for latest PCI device descriptions
+    try:
+        logging.info("Updating pci.ids for latest PCi device descriptions.")
```
Comment (gvancuts) [inline misc/config_tools/board_inspector/board_inspector.py:45]:
```suggestion
        logging.info("Updating pci.ids for latest PCI device descriptions.")
```

File: misc/config_tools/board_inspector/board_inspector.py [hunk 2]
```diff
@@ -40,6 +40,16 @@ def check_deps():
...
+        res = subprocess.Popen(["update-pciids", "-q"])
+        if res.wait() != 0:
+            logging.warning(f"Failed to invoke update-pciids. No functional impact is foreseen, but descriptions of PCI devices may be inaccurate.")
+            sys.exit(1)
+    except Exception as e:
+        logging.warning(f"Failed to invoke update-pciids: {e}. No functional impact is foreseen, but descriptions of PCI devices may be inaccurate.")
```
Comment (gvancuts) [inline misc/config_tools/board_inspector/board_inspector.py:51]:
```suggestion
        logging.warning(f"Failed to invoke update-pciids: {e}. No functional impact is foreseen, but descriptions of PCI devices may be unavailable.")
```

**Review by gvancuts — COMMENTED** [review]
This is harmless but also redundant to do both a `update-pciids` **and** `lspci -q`.

**Comment by wenlingz** [comment]
@NanlinXie please help approve this PR, thanks

**Review by NanlinXie — APPROVED** [review]

**Pull request merged**